### PR TITLE
Sun path in relation to day of the year

### PR DESCRIPTION
**Pull request by FlorianJanosch**

### CODE REVIEW

File: client/src/daytime.rs [hunk 1]
```diff
@@ -82,12 +87,19 @@ impl DayTime {
...
         let half_day = DAY_LENGTH as f32 / 2.0;
 
         let theta;
         let phi;
+
+        let mut month_diff = self.time_day as f32 - half_year;//
+        if month_diff < 0.0 {
+            //
+            month_diff *= -1.0//
+        }//
```
Comment (LukasKalbertodt) [inline client/src/daytime.rs]: Empty comments?

File: client/src/daytime.rs [hunk 2]
```diff
@@ -32,7 +32,12 @@ impl Default for DayTime {
 const DAY_LENGTH: f32 = 720.0;
 const YEAR_LENGTH: u32 = 12;
 
-const MONTH_DIFFERENCE: f32 = 3.1415 / 18.0;
+
+// lengthens the day by a static offset
+const DAY_LENGTHER: f32 = 0.0;
```
Comment (LukasKalbertodt) [inline client/src/daytime.rs]: Is this constant even used? Zero doesn't make sense to me...

File: client/src/daytime.rs [hunk 3]
```diff
@@ -82,12 +87,19 @@ impl DayTime {
     /// returns the position of the sun corresponding to time
     /// only mid summer
     pub fn get_sun_position(&self) -> Vector3f {
-        // 0 degrees for mid summer
-        // 60 degrees for hard winter
+
+        let half_year = YEAR_LENGTH as f32 / 2.0; //
```
Comment (LukasKalbertodt) [inline client/src/daytime.rs]: Empty comment?

File: client/src/daytime.rs (hunk 2 above)
Comment (FlorianJanosch) [inline client/src/daytime.rs]: This constant could be used for a static offset to lengthen the day later, but it could also be removed

**Pull request merged**